package.json

```diff
@@ -10,17 +10,25 @@
     "node": ">=18.0.0"
   },
   "scripts": {
+    "prepare": "husky",
     "publish-modules": "npx lerna run build && npx lerna publish",
     "build": "npx lerna run build",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx --fix && prettier --write \"**/*.{js,jsx,ts,tsx,json}\" && yarn constraints --fix",
+    "lint:js": "eslint . --ext .js,.jsx,.ts,.tsx --fix && prettier --write \"packages/**/*.{js,jsx,ts,tsx,json}\" && yarn constraints --fix",
+    "lint:clang": "bash ./scripts/run-clang-format.sh",
```

Comment:
minor nit but for consistency with scripts like `run-ios-tests.sh` you can remove bash here and add `#!/bin/bash` as the first line in the script, then can just have `./scripts/run-clang-format.sh`here